src/content/app/species/components/species-title-area/SpeciesTitleArea.tsx

```diff
@@ -70,6 +70,7 @@ const useSpecies = () => {
 };
 
 const SpeciesTitleArea = () => {
+  const activeGenomeId = useAppSelector(getActiveGenomeId) || '';
```

Comment:
There is no need to assign an empty string when `activeGenomeId` is null since that's been checked at line 83 below.